cmd/blockstorage_show.go

```diff
@@ -0,0 +1,91 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/spf13/cobra"
+
+	"github.com/exoscale/cli/pkg/globalstate"
+	"github.com/exoscale/cli/pkg/output"
+	v3 "github.com/exoscale/egoscale/v3"
+)
+
+type blockstorageShowOutput struct {
+	BlockStorageSnapshots []v3.BlockStorageSnapshotTarget `json:"block-storage-snapshots"`
+	Blocksize             int64                           `json:"blocksize"`
+	CreatedAT             time.Time                       `json:"created-at"`
+	ID                    v3.UUID                         `json:"id"`
+	Instance              *v3.InstanceTarget              `json:"instance"`
+	Labels                v3.Labels                       `json:"labels"`
+	Name                  string                          `json:"name"`
+	Size                  string                          `json:"size"`
+	State                 v3.BlockStorageVolumeState      `json:"state"`
+}
+
+func (o *blockstorageShowOutput) Type() string { return "Block Storage Volume" }
+func (o *blockstorageShowOutput) ToJSON()      { output.JSON(o) }
+func (o *blockstorageShowOutput) ToText()      { output.Text(o) }
+func (o *blockstorageShowOutput) ToTable()     { output.Table(o) }
+
+type blockstorageShowCmd struct {
+	cliCommandSettings `cli-cmd:"-"`
+
+	_ bool `cli-cmd:"show"`
+
+	Name string      `cli-arg:"#" cli-usage:"NAME|ID"`
+	Zone v3.ZoneName `cli-short:"z" cli-usage:"block storage volume zone"`
+}
+
+func (c *blockstorageShowCmd) cmdAliases() []string { return gShowAlias }
+
+func (c *blockstorageShowCmd) cmdShort() string { return "Show a Block Storage Volume details" }
+
+func (c *blockstorageShowCmd) cmdLong() string {
+	return fmt.Sprintf(`This command shows a Block Storage Volume details.
```

Comment:
```suggestion
	return fmt.Sprintf(`This command shows the details of a Block Storage Volume.
```

Reply:
Same here: https://github.com/exoscale/cli/blob/master/cmd/instance_show.go#L62
should we keep the same behavior?